mongodb_atlas/CHANGELOG.md

```diff
@@ -2,5 +2,8 @@
 
 ## 1.0.0 / 2023-03-21
 
-* [Added] Added changelog
+***Added***: 
+
+* Added changelog
```

Comment:
Can we change this to say `initial release`? I know this wasn't your writing ^, but its not really a changelog as it is haha.